ts/data/case.ts

```diff
@@ -187,7 +187,31 @@ export const caseDefinition: TableDefinition<Case, Test> = {
     }
     return null;
   },
-  staticActions: [legendAction],
+  staticActions: [
+    legendAction,
+    {
+      title: "TAT Report",
+      handler: (filters: { key: string; value: string }[]) => {
+        const currentFilters: { [key: string]: string[] } = {};
+        for (const filter of filters) {
+          if (filter.value !== undefined && filter.value !== null) {
+            if (!currentFilters[filter.key]) {
+              currentFilters[filter.key] = [];
+            }
+            currentFilters[filter.key].push(filter.value);
+          }
+        }
+        const joinedFilters: { [key: string]: string } = {};
+        for (const key in currentFilters) {
+          joinedFilters[key] = currentFilters[key].join(",");
+        }
```

Comment:
* Need to include the base filter here
* What if the filter value contains a comma?